README.md

```diff
@@ -17,6 +17,18 @@ When all the parts are finalized,
 
 When those parts are in place, the [docs](docs/) are updated, and we're ready to validate submissions.
 
+### Stability
+
+Once approved, both the list of metadata fields and the list of files is fixed.
+The metadata for a particular assay type needs to be consistent for all datasets,
+as does the set of files which comprise a dataset.
+Edits to descriptions are welcome, as are improved validations.
+
+For questions about the stability of metadata,
+contact Nils Gehlenborg, or add him as a reviewer on the PR.
```

Comment:
Who is this information for? TMCs? HIVE members? 

If outside HIVE, I would recommend that we do this in a centralized manner. We should have a single point of contact for the curation and ingestion process. This could be handled via the helpdesk, via Chris, or via Alex. 

I would propose that we establish a simple process that involves (1) collection of input at the point of contact and followed by (2) presentation of incoming metadata and directory structure requests on the portal call. 

@pdblood What do you think about this?

Reply:
> Who is this information for? TMCs? HIVE members?

It describes a process that will be followed by the HIVE when requests for changes come from TMCs. 

My goal is to remove myself from decision-making position... but to articulate a process so decisions still get made. I had thought it had buy-in from both of you... I've operationalized it by making issue templates that describe what will happen after someone submits an issue.

directories:
> ropelews will evaluate this request, and if it can move forward, will file a PR for pdblood to review, and mccalluc will make sure tests pass before merging.

metadata:
> cebriggs7135 will evaluate this request, and if it can move forward, will file a PR for ngehlenborg to review, and mccalluc will make sure tests pass before merging.

Let me know when the direction is clear.